lib/broadway_dashboard.ex

```diff
@@ -44,6 +44,7 @@ defmodule BroadwayDashboard do
       pipeline_config == :auto_discover ->
         case check_broadway_version(node) do
           :ok ->
+            # TODO: fix case when pids are returned
```

Comment:
Just a filter `is_atom()` should be enough. That would discard the recently added {:via, ...} tuples but I think it is fine too.

Reply:
cool, I will fix that in another PR :)